package.json

```diff
@@ -18,14 +18,27 @@
     "clean": "git clean -fxd",
     "build:bindings": "cmake-js configure -a x64 --CDCMAKE_PREFIX_INSTALL=distribute && cmake --build build --target install --config Release",
     "build:javascript": "tsc -p js",
-    "build:docs": "typedoc --out docs"
+    "build:docs": "typedoc --out docs",
+    "local:config": "yarn install && git submodule update --init --recursive --force && cmake -Bbuild -H. -G\"Visual Studio 15 2017\" -A x64 -DCMAKE_INSTALL_PREFIX=\"./obs-studio-node\"",
+    "local:build": "cmake --build build --target install",
```

Comment:
--config RelWithDebInfo/Debug is missing here.

Reply:
It uses Debug by default if --config is not used. Still want me to add it there?

Reply:
I would make it explicit, doesn't really hurt and prevent some potential mistakes.

Reply:
Made it explicit in the latest commit